### PR TITLE
Drop emissive on old Intel GPUs

### PR DESCRIPTION
**Pull request by Geenz**

### CODE REVIEW

**Review by RunitaiLinden — DISMISSED** [review]
Definitely too risky to go into ExtraFPS right away, might need to wait for a hotfix release. 
 Let's dogfood it for a day tomorrow and loop in kyle/atlas on if we want some QA on it before it merges down.

**Review by brad-linden — APPROVED** [review]

**Pull request merged**
